apps/studio/components/interfaces/ProjectCreation/PostgresVersionSelector.tsx

```diff
@@ -58,11 +58,16 @@ export const PostgresVersionSelector = ({
     data,
     isLoading: isLoadingProjectVersions,
     isSuccess,
-  } = useProjectCreationPostgresVersionsQuery({
-    cloudProvider,
-    dbRegion,
-    organizationSlug,
-  })
+  } = useProjectCreationPostgresVersionsQuery(
+    {
+      cloudProvider,
+      dbRegion,
+      organizationSlug,
+    },
+    {
+      refetchOnWindowFocus: false,
+    }
```

Comment:
this one seems unnecessary? the ticket that we're addressing is only about the region selector

postgres version selector is currently not available to users too, only internal